src/__test__/components/data-management/ProjectCard.test.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import configureMockStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import moment from 'moment';
+import initialState, { projectTemplate } from '../../../redux/reducers/projects/initialState';
+import ProjectCard from '../../../components/data-management/ProjectCard';
+
+const projectUuid = '12345';
+const projectName = 'Test Project';
+const samplesIdsArray = new Array(13).fill(null).map((_, i) => (`sample-${i}`));
+const createdDate = '2021-09-01T00:00:00.000Z';
+const lastAnalyzed = '2021-09-02T00:00:00.000Z';
+const lastModified = '2021-09-03T00:00:00.000Z';
+
+const projectState = {
+  projects: {
+    ...initialState,
+    experiments: [],
+    [projectUuid]: {
+      ...projectTemplate,
+      name: projectName,
+      uuid: projectUuid,
+      samples: samplesIdsArray,
+      createdDate,
+      lastAnalyzed,
+      lastModified,
+    },
+  },
+};
+
+const mockStore = configureMockStore([thunk]);
+
+describe('ProjectCard', () => {
```

Comment:
after you move the functionality of `ProjectCard` to be within the component itself, you will need to move the unit tests from `ProjectsList` to here.